observability/cockpit/how-to/send-metrics-logs-to-cockpit.mdx

```diff
@@ -30,8 +30,16 @@ This page shows you how to send metrics and logs to your [Cockpit](/observabilit
     <Message type="important">
      Once you have created your token, your secret key displays only once. Make sure you copy and save it before you exit the page.
     </Message>
-4. Configure your applications to send metrics and logs to your Cockpit endpoints, using your newly-created token for authentication.
-    <Lightbox src="scaleway_endpoints.webp" alt="" />
+4. Configure your applications to send metrics and logs to your Cockpit [endpoints](/observability/cockpit/concepts/#endpoints), using your newly-created token for authentication.
```

Comment:
Main issue with our client is that they don't know which endpoint to use to push metrics so for the 4th section I propose something like:
The observability product implement Remote Write endpoints for you to push metrics and logs.
For metrics the Remote Write endpoint is https://metrics.prd.obs.fr-par.scw.cloud/api/prom/push
We recommend you using Prom agent or grafana agent to push your metrics (link to specific documentation)
For logs the Remote Write endpoint is https://logs.prd.obs.fr-par.scw.cloud/loki/api/v1/push
We recommend you using Promtail or fluentbit agent to push your logs (link to specific documentation)